docs/adrs/0016-government-cloud-comparison.md

```diff
@@ -22,9 +22,16 @@ also manages.
 don't have what you want.
 - [Standaard Platform (Logius)](https://www.logius.nl/domeinen/infrastructuur/standaard-platform)
     - Description:
-        - The standard Platform of Logius will provide an Openshift Kubernetes namespace for you
+        - The standard Platform of Logius will provide an Openshift Kubernetes namespace or cluster for you
     - Pros:
+        - High availability is possible with an SLA (so production ready)
+        - More separation of concerns if you want to have a separate cluster
+        - Gitlab is included therefore easy CI/CD
+        - Convenient connection with the
+[Centraal Aansluitpunt](https://www.logius.nl/domeinen/infrastructuur/centraal-aansluitpunt) of Logius
     - Cons:
```

Comment:
```suggestion
    - Cons:
        - It is not vanilla Kubernetes
```

Reply:
yeah good point, i will also add these to digilab then

Reply:
As a Pro, you mean?